tests/ops/qutrit/test_qutrit_non_parametric_ops.py

```diff
@@ -105,10 +115,23 @@ def test_period_three_noninteger_power(self, op):
         with pytest.raises(qml.operation.PowUndefinedError):
             op.pow(1.234)
 
+    @pytest.mark.parametrize("offset", [0, 2, -2, 4, -4])
+    def test_tswap_pow(self, offset):
+        """Test powers of the TSWAP operator"""
+        op = qml.TSWAP(wires=[0, 1])
+
+        assert len(op.pow(0 + offset)) == 0
+        assert op.pow(1 + offset)[0].__class__ is qml.TSWAP
+
+        with pytest.raises(qml.operation.PowUndefinedError):
+            op.pow(1.234)  # Expect error raised for non-integer power
+
 
 label_data = [
     (qml.TShift(0), "TShift", "TShift⁻¹"),
     (qml.TClock(0), "TClock", "TClock⁻¹"),
+    (qml.TAdd([0, 1]), "TAdd", "TAdd⁻¹"),
+    (qml.TSWAP([0, 1]), "TSWAP", "TSWAP"),
```

Comment:
Is the swap left the same because swap^-1 = swap?

Reply:
Yep